src/Sylius/Behat/Page/Admin/ShippingMethod/IndexPageInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\ShippingMethod;
+
+use Sylius\Behat\Page\Admin\Crud\IndexPageInterface as BaseIndexPageInterface;
+
+/**
+ * @author Jan Góralski <jan.goralski@lakion.com>
+ */
+interface IndexPageInterface extends BaseIndexPageInterface
+{
+    /**
+     * @param string $isArchival
+     */
+    public function chooseArchival($isArchival);
+
+    /**
+     * @return bool
+     */
+    public function isArchival();
```

Comment:
isArchivalFilterEnabled? Or something along these lines, because is page archival ... hmmm, kinda make sense, but can be confusing as well.